SampleApps/WebView2WpfBrowser/MainWindow.xaml.cs

```diff
@@ -127,7 +160,7 @@ public MainWindow()
             InitializeComponent();
             AttachControlEventHandlers(webView);
             // Set background transparent
-            // webView.DefaultBackgroundColor = System.Drawing.Color.Transparent;
```

Comment:
Can we add those back before we have a solution on Win7? (We don't support transparent background on Win7.)